ci-operator/config/redhat-developer/rhdh/redhat-developer-rhdh-main.yaml

```
@@ -155,6 +155,11 @@ tests:
     - ref: redhat-developer-rhdh-osd-gcp-nightly
     workflow: redhat-developer-rhdh-osd-gcp-claim-cluster
   timeout: 3h40m0s
+- as: e2e-tests-nightly-auth-providers
+  cron: 0 7 * * *
```

If we want to be able to run the auth providers job from a PR check, you can add this and generate the presubmit job.
```suggestion
  cron: 0 7 * * *
  optional: true
  presubmit: true
```
Note that after the job is generated by the `make` commands, make sure to change the `always_run: true` to `always_run: false` for your job so it doesn't trigger automatically on new PRs

---

We are not adding the PR trigger at the moment, as this job will run a static cluster due to the Github redirect urls limitations.

---

I'll have a Jira anyway to track the PR checks trigger work. We will definitely need that but will require more work.